geps/gep-3792/index.md

```diff
@@ -0,0 +1,190 @@
+# GEP-3792: External Gateways
+
+* Issue: [#3792](https://github.com/kubernetes-sigs/gateway-api/issues/3792)
+* Status: Provisional
+
+(See [status definitions](../overview.md#gep-states).)
+
+## User Story
+
+**[Chihiro] and [Ian] want a way for out-of-cluster Gateways to be able to
+usefully participate in a GAMMA-compliant in-cluster service mesh.**
+
+Historically, API gateways and ingress controllers have often been implemented
+using a Service of type LoadBalancer fronting a Kubernetes pod running a
+proxy. This is simple to reason about, easy to manage for sidecar meshes, and
+will presumably be an important implementation mechanism for the foreseeable
+future. Some cloud providers, though, are moving the proxy outside of the
+cluster, for various reasons which are out of the scope of this GEP. Chihiro
+and Ian want to be able to use these out-of-cluster proxies effectively and
+safely, though they recognize that this may require additional configuration.
+
+[Chihiro]: https://https//gateway-api.sigs.k8s.io/concepts/roles-and-personas/#chihiro
+[Ian]: https://https//gateway-api.sigs.k8s.io/concepts/roles-and-personas/#ian
+
+### Nomenclature and Background
+
+In this GEP, we will use _out-of-cluster Gateway_ (OCG) to refer to a
+conformant implementation of Gateway API's `GATEWAY` profile that's running
+outside of the cluster. This would most commonly be a managed implementation
+from a cloud provider, but of course there are many other possibilities.
+
+We'll also distinguish between _mTLS meshes_, which rely on standard mTLS for
+secure communications including workload authentication, and _non-mTLS
+meshes_, which do anything else. We'll focus on mTLS meshes in this GEP; this
+isn't because of a desire to exclude non-mTLS meshes, but because we'll have
+enough trouble just wrangling the mTLS meshes! Supporting non-mTLS meshes will
+be a separate GEP.
```

Comment:
Just from the discussion I've seen I would propose that even supporting mTLS meshes would need to be a separate GEP.   Including the resolution of what constitutes an mTLS mesh seems like it could be handled separately.

Reply:
My intention was to continue in this GEP for the initial API, but maybe I'm misunderstanding @shaneutt's intent with this process...